sdcm/sct_config.py

```diff
@@ -1680,3 +1680,12 @@ def dump_help_config_yaml(self):
             ret += "{help_text}{name}: {default}\n\n".format(help_text=help_text, default=default, **opt)
 
         return ret
+
+
+def init_sct_config_from_environment() -> SCTConfiguration:
```

Comment:
better name will be `init_and_verify_sct_config`

Reply:
done